src/controllers/usersController.js

```diff
@@ -81,6 +82,77 @@ UsersController.prototype.getUser = function(req, res){
     }
 };
 
+
+
+
+
+
+
+
+
+/**
+ * @fn getAllUsers
+ * @desc Sends back the usernames of all users
+ * @param req Incoming message
+ * @param res Server Response
+ */
+UsersController.prototype.getAllUsers = function(req, res){
+    let _this = this;
+    let eaeUsername = req.body.eaeUsername;
+    let userToken = req.body.eaeUserToken;
+
```

Comment:
Maybe add another fields to be able to filter between all users, standard users and admin users?